internal/dynamicparams/database.ps1

```diff
@@ -70,4 +70,4 @@ $ScriptBlock = {
 }
 
 Register-DbaTeppScriptblock -ScriptBlock $ScriptBlock -Name Database
-Register-DbaTeppScriptblock -ScriptBlock $ScriptBlock -Name Exclude
\ No newline at end of file
+Register-DbaTeppScriptblock -ScriptBlock $ScriptBlock -Name ExcludeDatabase
```

Comment:
Scriptblocks need only be registered once and are named for the kind of objects they complete, not what is being done with them afterwards, so kill the second line, it's not being used.